.github/workflows/ci.yml

```diff
@@ -68,21 +72,22 @@ jobs:
     needs: [pre-commit]
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v3
 
     - name: Build sdist and wheel
       run: pipx run build
 
-    - uses: actions/upload-artifact@v2
+    - uses: actions/upload-artifact@v3
       with:
         path: dist
 
     - name: Check products
       run: pipx run twine check dist/*
 
     - uses: pypa/gh-action-pypi-publish@v1.5.0
-      if: github.event_name == 'release' && github.event.action == 'published'
+      if: github.event_name == 'release' && github.event.action == 'published' && github.repository == 'gradhep/relaxed'
       with:
         user: __token__
         # Remember to generate this and set it in "GitHub Secrets"
         password: ${{ secrets.pypi_password }}
+        print_hash: true
```

Comment:
what does `print_hash` do?

Reply:
@phinate From the [`v1.5.0` release notes](https://github.com/pypa/gh-action-pypi-publish/releases/tag/v1.5.0):

> Added an action input `print_hash` for showing the hash values of files to be uploaded